src/Presentation/FacetUiBuilder.php

```diff
@@ -181,6 +176,13 @@ private function getItemsHtml( array $items, string $type, string $propertyId ):
 		return $html;
 	}
 
+	private function facetTypeToTemplateName( FacetType $type ): string {
+		return match ( $type ) {
+			FacetType::LIST => 'FacetItemCheckbox',
+			FacetType::RANGE => 'FacetItemRange',
+		};
+	}
```

Comment:
@alistair3149 this is a more functional approach. No field and no $template variable